eth/backend.go

```diff
@@ -214,6 +215,8 @@ func New(stack *node.Node, config *ethconfig.Config) (*Ethereum, error) {
 		go eth.blockchain.StartFinalityVoteMonitor()
 	}
 
+	StartENRFilter(eth.blockchain, eth.p2pServer)
```

Comment:
Could we keep StartENRFilter in discovery.go seem this function belongs to discovery logic?  only move out logic from Start to New 

```
eth. StartENRFilter(eth.blockchain, eth.p2pServer)
```

Reply:
The problem is that in the New function, there is a variable named `eth`, so we cannot access the `eth` package in discovery.go

Reply:
Renaming the variable can work, but seems we need to replace it many times.
```
err := n.openEndpoints()
```
The p2p server will start after this entry, so I think the PR will solve this issue. LGTM